apps/backend/src/modules/class/formatter.ts

```diff
@@ -36,9 +49,14 @@ export const formatClass = (_class: IClassItem) => {
     primarySection: null,
     sections: null,
     gradeDistribution: null,
-  } as IntermediateClass;
-
-  return output;
+    aggregatedRatings: null,
+
+    description: _class.description,
+    gradingBasis: _class.gradingBasis as ClassGradingBasis,
+    finalExam: _class.finalExam as ClassFinalExam,
+    title: _class.title,
```

Comment:
`description` and `title` are filled in from `..._class` above. 

For `gradingBasis` and `finalExam`, could we move the enum types into the `models/class.ts`?